libwhatami/__init__.py

```diff
@@ -32,3 +32,7 @@ def main():
 
     with open(ascii_path) as ascii_file:
         print(ascii_file.read())
+
+def plainPrint():
```

Comment:
Cool! This is much simpler than what I was thinking.

Reply:
Thanks!